server/api.js

```diff
@@ -43,4 +43,24 @@ router.get("/laptop_request", async (req, res) => {
 	}
 });
 
+router.post("/laptop_donation", (req, res) => {
+	// create a database table for 
+	let name = req.body.name;
+	let address = req.body.address;
+	let laptops = req.body.numberOfLaptops;
```

Comment:
doesn't matter tremendously, but keep numberOfLaptops rather than laptops (which would be a good name for an array of laptops)